schemas/input_json_format.json

```diff
@@ -0,0 +1,143 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "http://example.com/example.json",
+  "type": "object",
+  "title": "The root schema",
+  "description": "https://github.com/vowpalwabbit/vowpal_wabbit/wiki/JSON",
+  "default": {},
+  "examples": [],
+  "required": [],
+  "definitions": {
+  },
+  "additionalProperties": true,
+  "patternProperties": {
+    "^[^_].*": {
+      "type": [
+        "string",
+        "number",
+        "boolean",
+        "array",
+        "object"
+      ],
+      "items": {
+        "type": "number"
+      }
+    },
+    "^_": {
+      "type": [
+        "number",
+        "string",
+        "boolean",
+        "object",
+        "array",
+        "null"
+      ]
+    }
+  },
+  "properties": {
+    "_tag": {
+      "$id": "#/properties/_tag",
+      "type": "string",
+      "title": "The _tag schema",
+      "description": "An explanation about the purpose of this instance.",
+      "default": "",
+      "examples": [
+        "mytag"
+      ]
+    },
+    "_label": {
```

Comment:
label might need to be converted to a definition for it to be referenced inside _multi but haven't fleshed out the requirements for _multi